routing/index_graph_starter.cpp

```diff
@@ -43,14 +43,23 @@ IndexGraphStarter::IndexGraphStarter(FakeEnding const & startEnding,
   : m_graph(graph)
 {
   m_startId = fakeNumerationStart;
+  m_startPassThroughAllowed = EndingPassThroughAllowed(startEnding);
   AddStart(startEnding, finishEnding, strictForward, fakeNumerationStart);
   m_finishId = fakeNumerationStart;
   AddFinish(finishEnding, startEnding, fakeNumerationStart);
+  m_finishPassThroughAllowed = EndingPassThroughAllowed(finishEnding);
```

Comment:
Please, swap this line with the previous one, for consistency with start initialization.

Reply:
moved to initialization list